aten/src/ATen/native/cpu/zmath.h

```diff
@@ -35,7 +35,10 @@ inline double zabs <c10::complex<double>, double> (c10::complex<double> z) {
 
 template <typename SCALAR_TYPE, typename VALUE_TYPE=SCALAR_TYPE>
```

Comment:
Add a comment explaining this function

Reply:
I think the name `argument/arg` is the more appropriate term from complex number perspective. What do you think? 
But `angle` also makes sense as the user facing function is called angle.
About `angle_impl`, all the functions in `zmath.h` consistently have `_impl` suffix.

https://github.com/pytorch/pytorch/blob/76f7b733dd169c29a96fd8c0c4671a6cd6ba2818/aten/src/ATen/native/cpu/zmath.h#L114-L163

Reply:
I guess we're stuck with "_impl", then. If the name is consistent then this PR doesn't need to bother updating it. But a comment explaining it would still be good.

Reply:
Have added the following comment.

https://github.com/pytorch/pytorch/pull/49163/files#diff-de6356355016a3f8ef55d44f8252b42fd81fca56421b6097427feb1b7a63a5ebR36-R41